pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1336,6 +1336,19 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 	return causes
 }
 
+func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, vmi *v1.VirtualMachineInstance) []metav1.StatusCause {
```

Comment:
validating-webhook_test.go needs an entry to ensure this validation is enforced.